src/helm/benchmark/adaptation/adapter_spec.py

```diff
@@ -68,6 +68,9 @@ class AdapterSpec:
     # set of training instances.  Used to compute error bars.
     num_train_trials: int = 1
 
+    # If true, randomly sample N training examples; if false, select the first N training examples
```

Comment:
"if false, select N consecutive training examples" (since it's not always the first)